eng/templates/steps/docker-build.yml

```diff
@@ -6,6 +6,21 @@ parameters:
   type: string
 
 steps:
+- script: |
+    . ./eng/common/tools.sh
+    InitializeDotNetCli true
+    version=$(./.dotnet/dotnet msbuild \
+      ./src/ProductConstructionService/ProductConstructionService.Api/ProductConstructionService.Api.csproj \
```

Comment:
so we're building the api just to see what package version we'll get? If that's the case maybe we should build something smaller?

Reply:
We're not, we just do `getProperty` on it, so it evaluates everything but does not run any target

Reply:
ahh ok, some msbuild magic :D